test-scripts/test-results/.env.example

```
@@ -0,0 +1,17 @@
+POSTGRES_URL=postgresql://postgres:postgres@localhost:5555/eliza2
+
+LOG_LEVEL=debug
+
+OPENROUTER_API_KEY=sk-or-v1-api-key
+# ANTHROPIC
+OPENROUTER_SMALL_MODEL=anthropic/claude-3.5-sonnet-20240620
+OPENROUTER_LARGE_MODEL=anthropic/claude-3.5-sonnet-20240620
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The OPENROUTER_LARGE_MODEL key should go before the OPENROUTER_SMALL_MODEL key

(UnorderedKey)